Sources/BraveWallet/Panels/Signature Request/SignMessageRequestView.swift

```diff
@@ -0,0 +1,400 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveStrings
+import BraveCore
+import DesignSystem
+
+/// View for showing `SignMessageRequest` for 
+/// ethSignTypedData, ethStandardSignData, & solanaSignData
+struct SignMessageRequestView: View {
+  
+  let account: BraveWallet.AccountInfo
+  let request: BraveWallet.SignMessageRequest
+  let network: BraveWallet.NetworkInfo?
+  let requestIndex: Int
+  let requestCount: Int
+  /// A map between request id and a boolean value indicates this request message needs pilcrow formating.
+  @Binding var needPilcrowFormatted: [Int32: Bool]
+  /// A map between request id and a boolean value indicates this request message is displayed as
+  /// its original content.
+  @Binding var showOrignalMessage: [Int32: Bool]
+  var nextTapped: () -> Void
+  var action: (_ approved: Bool) -> Void
+  
+  @Environment(\.sizeCategory) private var sizeCategory
+  @ScaledMetric private var blockieSize = 54
+  private let maxBlockieSize: CGFloat = 108
+  private let staticTextViewHeight: CGFloat = 200
+  
+  /// Request display text, used as fallback.
+  private var requestDisplayText: String {
+    if requestDomain.isEmpty {
+      return requestMessage
+    }
+    return """
+    \(Strings.Wallet.signatureRequestDomainTitle):
+    \(requestDomain)
+    
+    \(Strings.Wallet.signatureRequestMessageTitle):
+    \(requestMessage)
```

Comment:
Colon's are considered punctuation and may be slightly different in different languages, so it should be localized as well. You may need to  alter `signatureRequestDomainTitle`/`signatureRequestMessageTitle` to include the colon and a `%@` for the value then format it in